### PR TITLE
Remove calls to already implemented functions

### PR DESCRIPTION
**Pull request by marijnvdwerf**

### CODE REVIEW

**Review by duncanspumpkin — APPROVED** [review]
All looks good

File: src/OpenLoco/Windows/TitleVersion.cpp [hunk 1]
```diff
@@ -30,7 +30,6 @@ namespace OpenLoco::Ui::Windows::TitleVersion
             &_events);
         window->widgets = widgets;
 
-        _events.prepare_draw = (void (*)(Ui::Window*))0x0042A035;
```
Comment (AaronVanGeffen) [inline src/OpenLoco/Windows/TitleVersion.cpp:33]: Oh wow. We set a vanilla routine to our own custom window? Amazing

File: src/OpenLoco/Windows/ToolbarTop.cpp [hunk 2]
```diff
@@ -154,7 +147,7 @@ namespace OpenLoco::Ui::Windows::ToolbarTop::Game
             return;
         }
 
-        if (!saveGameOpen())
+        if (!OpenLoco::Game::saveSaveGameOpen())
```
Comment (AaronVanGeffen) [inline src/OpenLoco/Windows/ToolbarTop.cpp:150]: Any particular reason for explicitly resolving `OpenLoco::`?

File: src/OpenLoco/Windows/TitleVersion.cpp (hunk 1 above)
Comment (duncanspumpkin) [reply src/OpenLoco/Windows/TitleVersion.cpp:33]: its just a ret

Comment (marijnvdwerf) [reply src/OpenLoco/Windows/TitleVersion.cpp:33]: (`prepare_draw` wasn't implemented yet this early on)

File: src/OpenLoco/Windows/ToolbarTop.cpp (hunk 2 above)
Comment (marijnvdwerf) [reply src/OpenLoco/Windows/ToolbarTop.cpp:150]: Just applying the suggestion clang gave in https://github.com/OpenLoco/OpenLoco/runs/3365202052?check_suite_focus=true

Comment (AaronVanGeffen) [reply src/OpenLoco/Windows/ToolbarTop.cpp:150]: Oh, right, `Game` is also a namespace in `ToolbarTop`. Carry on then.

**Review by AaronVanGeffen — APPROVED** [review]

File: src/OpenLoco/Windows/TerraForm.cpp [hunk 3]
```diff
@@ -2671,58 +2671,5 @@ namespace OpenLoco::Ui::Windows::Terraform
 
     void registerHooks()
     {
```
Comment (AaronVanGeffen) [inline src/OpenLoco/Windows/TerraForm.cpp:2673]: Could you remove this function as well, given it no longer contains any hooks?

**Pull request merged**